libr/core/cmd_print.c

```diff
@@ -13,11 +13,11 @@
 
 static const char *help_msg_pa[] = {
 	"Usage: pa[edD]", "[asm|hex]", "print (dis)assembled",
-	"pa", " [assembly]", "print hexpairs of the given assembled instruction",
-	"paD", " [assembly]", "print assembled from hexparis",
-	"pad", " [hexpairs]", "print disassembled from hexpairs",
-	"pae", " [assembly]", "print ESIL after assembling given instruction",
-	"pade", " [hexpairs]", "print ESIL expression assembled ESIL from hexpairs",
+	"pa", " [assembly]", "print hexpairs of the given assembly expression",
+	"pad", " [hexpairs]", "print assembly expression from hexpairs",
+	"paD", " [hexpairs]", "print assembly expression from hexpairs and show hexpairs",
```

Comment:
"print disassembly expression" looks wrong to me. something like "disassemble given hexpairs" is more concise imho

Reply:
I don't mind the terminology as long as it is consistent over all commands.
I thought combining {"assembled instruction", "disassembled", "after assembling given instruction"} to just "assembly expression" might be a good idea, since there was already the term "ESIL expression".

However, to keep it short & consistent, omitting "expression" and both verbs assemble/disassemble would probably be best.

What do you think about this suggestion:
```
pa [assembly]:   print hexpairs (asm -> hex)
pad [hexpairs]:  print assembly (hex -> asm)
paD [hexpairs]:  print assembly and hexpairs (hex -> asm & hex)
pade [hexpairs]: print ESIL (hex -> esil)
pae [assembly]:  print ESIL (asm -> esil)
```

Reply:
the shorter the better. btw "print hexpairs" is assemble, and "print assembly" is "disassemble"